src/main/java/com/infobip/interviewdemo/service/CodeReviewService.java

```diff
@@ -43,6 +47,7 @@ private CodeReviewDetailResponse mapEntityToResponse(CodeReviewEntity entity) {
                 .author(entity.getAuthor())
                 .reviewers(entity.getReviewers())
                 .state(entity.getState())
+                .commentsCount(commentsCount)
```

Comment:
```suggestion
                .commentsCount(entity.getCommentEntities().size())
```
but you would need to add mapping mentioned above